src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj

```diff
@@ -6,6 +6,6 @@
     <AssemblyName>Microsoft.Azure.Management.ServiceBus</AssemblyName>
     <PackageTags>Microsoft Azure ServiceBus Management;ServiceBus;ServiceBus management</PackageTags>      
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-    <VersionPrefix>1.0.1</VersionPrefix>
+    <VersionPrefix>1.0.2</VersionPrefix>
```

Comment:
Add < PackageReleaseNotes > and add Release notes.
I would recommend to use < ![CDATA and then format your Release notes

https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj#L10

Reply:
updated package release notes

Reply:
@shahabhijeet, have updated package release notes. can you please review it.